pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1685,6 +1696,7 @@ static __always_inline int init_event_data(event_data_t *data, void *ctx)
     if (unlikely(data->config == NULL))
         return 0;
 
+    data->task = (struct task_struct *) bpf_get_current_task();
```

Comment:
Is this duplicate on purpose?

Reply:
Nice catch, I believe it is a mistake. We could have it fixed when merging through github UI but then tests don't run when we do manual changes, so It is better to have it fixed and code repushed.

Reply:
Wow, can't believe I didn't notice this.
I fix it and push the new version.